docs/main/prerequisites.md

```diff
@@ -8,7 +8,7 @@ last_updated: May 25 2020
 
 Pi-hole is very lightweight and does not require much processing power
 
-- ~52MB of free space
+- ~52MB of free space (excl. long-term and gravity databases) 
```

Comment:
And repository packages, logs, general system overhead....

Better to just say "Min 4GB SD Card"

Reply:
You mean replace 52MB all together with "Min 4GB SD Card"

Reply:
Yeah, I think that's a fair minimum. Or say "2GB minimum, 4GB for all features"?

Reply:
Everything I'm seeing for prices has 2GB as more expensive than 16GB microSD.